fideslog/sdk/python/event.py

```diff
@@ -123,3 +123,43 @@ def __init__(
 
         except AssertionError as err:
             raise InvalidEventError(str(err)) from None
+
+
+class UserRegistrationEvent:
```

Comment:
Maybe rename this file to `analytics_event.py` and put this class into a new `user_registration_event.py` file?